scripts/core/editor3/actions/spellchecker.jsx

```diff
@@ -0,0 +1,13 @@
+export function replaceWord(word) {
+    return {
+        type: 'REPLACE_WORD',
+        payload: word
+    };
+}
+
+export function showContextMenu(contextMenuData) {
```

Comment:
The function is called `showContextMenu` and the parameter is `data`. It is obvious the data is for the context menu. This is again stuttering and only complicates things and makes code longer to read and write. I think it's about being simple, not specific.